tests/flows/teleporter_utils.go

```diff
@@ -0,0 +1,42 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package flows
+
+import (
+	"math/big"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	teleporterutils "github.com/ava-labs/teleporter/utils/teleporter-utils"
+	"github.com/ethereum/go-ethereum/common"
+	. "github.com/onsi/gomega"
+)
+
+// Tests Teleporter utility functions
```

Comment:
wasn't sure if this test is meant to test the other utils functions as well, but if not, this comment would still need updating